chapters/cpp_for_opencl.md

```diff
@@ -1,8 +1,8 @@
 # C++ for OpenCL
 
-The OpenCL working group has transitioned from the original OpenCL C++ kernel language first defined in OpenCL 2.1 to the community developed  [C++ for OpenCL](https://github.com/KhronosGroup/Khronosdotorg/blob/master/api/opencl/assets/CXX_for_OpenCL.pdf) open source front end compiler that provides improved features and compatibility with OpenCL C.
+The OpenCL working group has transitioned from the original OpenCL C++ kernel language first defined in OpenCL 2.1 to the community developed [C++ for OpenCL](https://www.iwocl.org/wp-content/uploads/iwocl-syclcon-2020-stulova-13-slides.pdf) kernel language that provides improved features and compatibility with OpenCL C.
```

Comment:
Minor nitpick: I think the OpenCL C++ kernel language wasn't defined until OpenCL 2.2?

Reply:
Changed! Thanks!